ACMG variant classification for WES

andyb3 opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/woook/moka-fe/431)
<!-- Reviewable:end -->

aledj2 commented:
Reviewed 5 of 9 files at r1.
Review status: 5 of 9 files reviewed at latest revision, 28 unresolved discussions.

---

*[forms/s0901_NGSCurrentVar, line 52 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LSWDlO3wrIGl2vllB:-L2LSWDlO3wrIGl2vllC:b69l9vw) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L52)):*
> Set rsAddComment = New ADODB.Recordset
' create empty recordset to add new comment

---

*[forms/s0901_NGSCurrentVar, line 53 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LSaG1w-HkEt-x31_W:-L2LSaG1w-HkEt-x31_X:b-mp6pi6) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L53)):*
> sqlAddComment = "INSERT INTO NGSVariantComment(Comment) VALUES ('')"
' SQL query to insert an empty comment into NGSVariantComment table

---

*[forms/s0901_NGSCurrentVar, line 54 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LSktnj7SVT-HLeJfn:-L2LSktnj7SVT-HLeJfo:bf3r7q7) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L54)):*
> rsAddComment.Open sqlAddComment, CurrentProject.Connection, adOpenKeyset
' execute the query to add empty comment

---

*[forms/s0901_NGSCurrentVar, line 56 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LSrf0i25C5zhCHTGa:-L2LSrf0i25C5zhCHTGb:b8fyg0g) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L56)):*
> commentID = rsAddComment.Fields(0).Value
' capture the commentID for this record to the key for the newly created empty comment

---

*[forms/s0901_NGSCurrentVar, line 57 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LT-ofGGqEHPfeu3g8:-L2LT-ofGGqEHPfeu3g9:b-c8x8s3) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L57)):*
> Set rsAddComment = Nothing
remove the recordset - BTW why is this done?

---

*[forms/s0901_NGSCurrentVar, line 58 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LTDKX8we0Oq6cIRoC:-L2LTDKX8we0Oq6cIRoD:bea8kfr) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L58)):*
>  Me.NGSVariantCommentID = commentID
' set the commentID for this record to the key for the newly created empty comment

---

*[forms/s0901_NGSCurrentVar, line 63 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LTcSX6ryelUaqN0hu:-L2LTcSX6ryelUaqN0hv:b81yosv) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L63)):*
> Set rsAddACMG = New ADODB.Recordset
' create new recordset

---

*[forms/s0901_NGSCurrentVar, line 64 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LTeT2lDrrS1uEUAWS:-L2LTeT2lDrrS1uEUAWT:bak8cpy) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L64)):*
> sqlAddACMG = "INSERT INTO NGSVariantACMG(NGSVariantID) VALUES (" & Me.NGSVariantID & ")"
sql query to insert NGSVariantID into NGSVariantACMG table

---

*[forms/s0901_NGSCurrentVar, line 65 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LTrsB4WsucZd-lFb9:-L2LTrsB4WsucZd-lFbA:bbk9q0x) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L65)):*
> rsAddACMG.Open sqlAddACMG, CurrentProject.Connection, adOpenKeyset
'execute the query

---

*[forms/s0901_NGSCurrentVar, line 66 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LTtuRGohRehugM7Q5:-L2LTtuRGohRehugM7Q6:bpn9ay9) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L66)):*
> Set rsAddACMG = Nothing
'empty the recordset

---

*[forms/s0901_NGSCurrentVar, line 69 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LU6J1dmt2GJSqDyYz:-L2LU6J1dmt2GJSqDyZ-:bqgiqni) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L69)):*
> ```
>     End If
>     
>     stDocName = "s0901_NGSVariantACMGPopup"
> ```

' define name of the for to open

---

*[forms/s0901_NGSCurrentVar, line 70 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LUA72_5d1xm0Cf2_b:-L2LUA72_5d1xm0Cf2_c:b-smspav) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L70)):*
> ```
>     
>     stDocName = "s0901_NGSVariantACMGPopup"
>     stLinkCriteria = "[NGSVariantID] = " & Me.NGSVariantID
> ```

set link between sub-form and this form

---

*[forms/s0901_NGSCurrentVar, line 81 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LUzSpcdbS01IZdbgU:-L2LUzSpcdbS01IZdbgV:bppxsjm) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L81)):*
> ```
> 
> Private Sub CommentShort_MouseDown(Button As Integer, Shift As Integer, x As Single, Y As Single)
>     If Shift = acCtrlMask Then
> ```

what does this do?

---

*[forms/s0901_NGSPrevVars, line 16 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVAyHDpIcszKwXTFT:-L2LVAyHDpIcszKwXTFU:b-axhqjs) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L16)):*
> ```
>     'If there isn't a comment, create an empty one.
>     If IsNull(Me.NGSVariantCommentID) Then
>         Set rsAddComment = New ADODB.Recordset
> ```

' create empty recordset

---

*[forms/s0901_NGSPrevVars, line 17 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVDtRmavzoT7AAYeB:-L2LVDtRmavzoT7AAYeC:b-o3ps16) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L17)):*
> ```
>     If IsNull(Me.NGSVariantCommentID) Then
>         Set rsAddComment = New ADODB.Recordset
>         sqlAddComment = "INSERT INTO NGSVariantComment(Comment) VALUES ('')"
> ```

'sql query to insert empty comment into NGSVariantComment

---

*[forms/s0901_NGSPrevVars, line 18 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVIwRZ9nyeGRrMGDb:-L2LVIwRZ9nyeGRrMGDc:b-cksxdd) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L18)):*
> ```
>         Set rsAddComment = New ADODB.Recordset
>         sqlAddComment = "INSERT INTO NGSVariantComment(Comment) VALUES ('')"
>         rsAddComment.Open sqlAddComment, CurrentProject.Connection, adOpenKeyset
> ```

'create recordset

---

*[forms/s0901_NGSPrevVars, line 20 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVM8ITF1oqrOIzZRA:-L2LVM8ITF1oqrOIzZRB:b-yqzhr7) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L20)):*
> ```
>         rsAddComment.Open sqlAddComment, CurrentProject.Connection, adOpenKeyset
>         rsAddComment.Open "SELECT @@identity", CurrentProject.Connection, adOpenKeyset 'Get ID of new entry
>         commentID = rsAddComment.Fields(0).Value
> ```

capture the commentid after inserting empty comment

---

*[forms/s0901_NGSPrevVars, line 21 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVR_M5tuQu4--ODHv:-L2LVR_M5tuQu4--ODHw:bxlasf0) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L21)):*
> ```
>         rsAddComment.Open "SELECT @@identity", CurrentProject.Connection, adOpenKeyset 'Get ID of new entry
>         commentID = rsAddComment.Fields(0).Value
>         Set rsAddComment = Nothing
> ```

'empty recordset

---

*[forms/s0901_NGSPrevVars, line 22 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVSyK0W_kmLSGcMJP:-L2LVSyK0W_kmLSGcMJQ:b-1sdh1i) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L22)):*
> ```
>         commentID = rsAddComment.Fields(0).Value
>         Set rsAddComment = Nothing
>         Me.NGSVariantCommentID = commentID
> ```

'capture the id of the newly imported comment into table

---

*[forms/s0901_NGSPrevVars, line 27 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVXM_Bi6Fmd5w0_aF:-L2LVXM_Bi6Fmd5w0_aG:b-hpbcwl) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L27)):*
> ```
>     'If there isn't an existing ACMG record for the variant, create an empty one
>     If IsNull(Me.NGSVariantACMGID) Then
>         Set rsAddACMG = New ADODB.Recordset
> ```

' create recordset

---

*[forms/s0901_NGSPrevVars, line 28 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVZSUSxL3012-CFSM:-L2LVZSUSxL3012-CFSN:b-snb4bq) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L28)):*
> ```
>     If IsNull(Me.NGSVariantACMGID) Then
>         Set rsAddACMG = New ADODB.Recordset
>         sqlAddACMG = "INSERT INTO NGSVariantACMG(NGSVariantID) VALUES (" & Me.NGSVariantID & ")"
> ```

' sql to create record for this variant

---

*[forms/s0901_NGSPrevVars, line 29 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVg5rJg58CX9LD2pf:-L2LVg5rJg58CX9LD2pg:b1tldcr) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L29)):*
> ```
>         Set rsAddACMG = New ADODB.Recordset
>         sqlAddACMG = "INSERT INTO NGSVariantACMG(NGSVariantID) VALUES (" & Me.NGSVariantID & ")"
>         rsAddACMG.Open sqlAddACMG, CurrentProject.Connection, adOpenKeyset
> ```

'execute query

---

*[forms/s0901_NGSPrevVars, line 30 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVhTc0ros_HiAYzBw:-L2LVhTc0ros_HiAYzBx:bxlasf0) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L30)):*
> ```
>         sqlAddACMG = "INSERT INTO NGSVariantACMG(NGSVariantID) VALUES (" & Me.NGSVariantID & ")"
>         rsAddACMG.Open sqlAddACMG, CurrentProject.Connection, adOpenKeyset
>         Set rsAddACMG = Nothing
> ```

'empty recordset

---

*[forms/s0901_NGSVariantACMGPopup, line 58 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LWM4wIJm75_i-PuVa:-L2LWM4wIJm75_i-PuVb:b-cksxdd) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSVariantACMGPopup#L58)):*
> ```
>     
>     'Query to get the 5 classifications from the Status table.
>     Set status_classification = New ADODB.Recordset
> ```

'create recordset

---

*[forms/s0901_NGSVariantACMGPopup, line 59 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LWO6zVlLE6Z650RTj:-L2LWO6zVlLE6Z650RTk:bcfouyw) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSVariantACMGPopup#L59)):*
> ```
>     'Query to get the 5 classifications from the Status table.
>     Set status_classification = New ADODB.Recordset
>     sql_status_classification = "SELECT TOP 5 Status.StatusID, Status.Status, Status.Pathogenic FROM Status WHERE Status.Pathogenic = 1 ORDER BY Status.SortingPathogenic;"
> ```

'sql to extract the 5 classifications from the status table

---

*[forms/s0901_NGSVariantACMGPopup, line 60 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LWSrEUbwlw7KEKeJt:-L2LWSrEUbwlw7KEKeJu:b-wtywwk) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSVariantACMGPopup#L60)):*
> ```
>     Set status_classification = New ADODB.Recordset
>     sql_status_classification = "SELECT TOP 5 Status.StatusID, Status.Status, Status.Pathogenic FROM Status WHERE Status.Pathogenic = 1 ORDER BY Status.SortingPathogenic;"
>     status_classification.Open sql_status_classification, CurrentProject.Connection, adOpenKeyset
> ```

'populate recordset with the classification

---

*[forms/s0901_NGSVariantACMGPopup, line 375 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LY2GqLNoXhaDcHGn0:-L2LY2GqLNoXhaDcHGn1:bylkrnz) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSVariantACMGPopup#L375)):*
> ```
>         rule_applied = "Likely benign (ii) >=2 Supporting (BP1–BP7)"
>     Else
>         MsgBox "ERROR"
> ```

could this be more informative?

---

*[forms/s0901_NGSVariants, line 23 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LZUm9duTY--0ucn4b:-L2LZUm9duTY--0ucn4c:b3rzctv) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSVariants#L23)):*
> ```
> Private Sub commentShort_DblClick(Cancel As Integer)
>     Dim stDocName As String
>     Dim rsAddComment As ADODB.Recordset
> ```

please add some comments to this

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/woook/moka-fe/431)*
<!-- Sent from Reviewable.io -->

andyb3 commented:
Review status: 5 of 9 files reviewed at latest revision, 29 unresolved discussions.

---

*[forms/s0901_NGSCurrentVar, line 52 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LSWDlO3wrIGl2vllB:-L2QBxig9txGsWhiH3Rb:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L52)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

> Set rsAddComment = New ADODB.Recordset
' create empty recordset to add new comment
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 53 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LSaG1w-HkEt-x31_W:-L2QBzCcvXc9OYgte53y:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L53)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

> sqlAddComment = "INSERT INTO NGSVariantComment(Comment) VALUES ('')"
' SQL query to insert an empty comment into NGSVariantComment table
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 54 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LSktnj7SVT-HLeJfn:-L2QC-39u_wXuHZ4iZkj:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L54)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

> rsAddComment.Open sqlAddComment, CurrentProject.Connection, adOpenKeyset
' execute the query to add empty comment
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 56 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LSrf0i25C5zhCHTGa:-L2QC0T-XvEdemIZr5Bx:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L56)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

> commentID = rsAddComment.Fields(0).Value
' capture the commentID for this record to the key for the newly created empty comment
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 57 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LT-ofGGqEHPfeu3g8:-L2QC1pweS1cqQFg2eQV:b73rllz) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L57)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

> Set rsAddComment = Nothing
remove the recordset - BTW why is this done?
</blockquote></details>

If you don't close the recordset it leaves a connection to the database open. Apparently in older versions of Access this could cause it to hang when you try to close the form, although don't think it's a problem any more.

---

*[forms/s0901_NGSCurrentVar, line 58 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LTDKX8we0Oq6cIRoC:-L2QDKNOn0cVfqm_1MtC:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L58)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

>  Me.NGSVariantCommentID = commentID
' set the commentID for this record to the key for the newly created empty comment
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 63 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LTcSX6ryelUaqN0hu:-L2QDM2b5EpUEkQIrRC1:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L63)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

> Set rsAddACMG = New ADODB.Recordset
' create new recordset
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 64 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LTeT2lDrrS1uEUAWS:-L2QDNlCnrE7RTzq19ns:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L64)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

> sqlAddACMG = "INSERT INTO NGSVariantACMG(NGSVariantID) VALUES (" & Me.NGSVariantID & ")"
sql query to insert NGSVariantID into NGSVariantACMG table
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 65 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LTrsB4WsucZd-lFb9:-L2QDTTzAniW4rd-PYxL:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L65)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

> rsAddACMG.Open sqlAddACMG, CurrentProject.Connection, adOpenKeyset
'execute the query
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 66 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LTtuRGohRehugM7Q5:-L2QDU6UD2l-GJ-As_6J:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L66)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

> Set rsAddACMG = Nothing
'empty the recordset
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 69 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LU6J1dmt2GJSqDyYz:-L2QDVBXEOa7xjaymG5R:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L69)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

' define name of the for to open
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 70 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LUA72_5d1xm0Cf2_b:-L2QDVsmv6iT-mxSED63:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L70)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

set link between sub-form and this form
</blockquote></details>

Done.

---

*[forms/s0901_NGSCurrentVar, line 81 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LUzSpcdbS01IZdbgU:-L2QDXsxv22SpAGbLEZY:b-n3e70p) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSCurrentVar#L81)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

what does this do?
</blockquote></details>

So I was meant to move this to be double click on the date field. I've now changed it and commented it

---

*[forms/s0901_NGSPrevVars, line 16 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVAyHDpIcszKwXTFT:-L2QDi6UrfNzEYcJQF-X:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L16)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

' create empty recordset
</blockquote></details>

Done.

---

*[forms/s0901_NGSPrevVars, line 17 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVDtRmavzoT7AAYeB:-L2QDiW1I5SsqbYVtJWB:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L17)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

'sql query to insert empty comment into NGSVariantComment
</blockquote></details>

Done.

---

*[forms/s0901_NGSPrevVars, line 20 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVM8ITF1oqrOIzZRA:-L2QDj8hrfEXxyZ8MbHn:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L20)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

capture the commentid after inserting empty comment
</blockquote></details>

Done.

---

*[forms/s0901_NGSPrevVars, line 21 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVR_M5tuQu4--ODHv:-L2QDk1lFyav26h5AaXi:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L21)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

'empty recordset
</blockquote></details>

Done.

---

*[forms/s0901_NGSPrevVars, line 22 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVSyK0W_kmLSGcMJP:-L2QDk_yerS2-0NWh345:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L22)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

'capture the id of the newly imported comment into table
</blockquote></details>

Done.

---

*[forms/s0901_NGSPrevVars, line 27 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVXM_Bi6Fmd5w0_aF:-L2QDmfn1cW3q0cXgQrw:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L27)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

' create recordset
</blockquote></details>

Done.

---

*[forms/s0901_NGSPrevVars, line 28 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVZSUSxL3012-CFSM:-L2QDnlxET25GGs-0d7b:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L28)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

' sql to create record for this variant
</blockquote></details>

Done.

---

*[forms/s0901_NGSPrevVars, line 29 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVg5rJg58CX9LD2pf:-L2QDonL2E5nayLffq-T:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L29)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

'execute query
</blockquote></details>

Done.

---

*[forms/s0901_NGSPrevVars, line 30 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVhTc0ros_HiAYzBw:-L2QDpE-_FV2YTlvCg6U:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L30)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

'empty recordset
</blockquote></details>

Done.

---

*[forms/s0901_NGSPrevVars, line 39 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2QDr3XVg9MIWWr-GO_:-L2QDr3XVg9MIWWr-GOa:b-ar8oua) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L39)):*
> ```
> End Sub
> 
> Private Sub CommentShort_MouseDown(Button As Integer, Shift As Integer, x As Single, Y As Single)
> ```

Updated this so it is double click on the date field

---

*[forms/s0901_NGSVariantACMGPopup, line 58 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LWM4wIJm75_i-PuVa:-L2QDxSOjvAh1evJKwxc:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSVariantACMGPopup#L58)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

'create recordset
</blockquote></details>

Done.

---

*[forms/s0901_NGSVariantACMGPopup, line 59 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LWO6zVlLE6Z650RTj:-L2QDy5quQMyTqG3SdZI:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSVariantACMGPopup#L59)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

'sql to extract the 5 classifications from the status table
</blockquote></details>

Done.

---

*[forms/s0901_NGSVariantACMGPopup, line 60 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LWSrEUbwlw7KEKeJt:-L2QDyY6jgKFfvzXq5mT:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSVariantACMGPopup#L60)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

'populate recordset with the classification
</blockquote></details>

Done.

---

*[forms/s0901_NGSVariantACMGPopup, line 375 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LY2GqLNoXhaDcHGn0:-L2Pzmg4uAsbPCWgb8Lu:b-wn1cky) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSVariantACMGPopup#L375)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

could this be more informative?
</blockquote></details>

Woops! That was just meant to be a placeholder. I have now replaced it with a proper message!

---

*[forms/s0901_NGSVariants, line 23 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LZUm9duTY--0ucn4b:-L2QE6MEgwStiejjI4ak:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSVariants#L23)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

please add some comments to this
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/woook/moka-fe/431)*
<!-- Sent from Reviewable.io -->

andyb3 commented:
Review status: 5 of 9 files reviewed at latest revision, 29 unresolved discussions.

---

*[forms/s0901_NGSPrevVars, line 18 at r1](https://reviewable.io:443/reviews/woook/moka-fe/431#-L2LVIwRZ9nyeGRrMGDb:-L2QEA8ePvS2tNt6abZS:b-896fix) ([raw file](https://github.com/woook/moka-fe/blob/5ed7902fbd80fef8e50e1b4ec999921c51e764f9/forms/s0901_NGSPrevVars#L18)):*
<details><summary><i>Previously, aledj2 (Aled Jones) wrote…</i></summary><blockquote>

'create recordset
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/woook/moka-fe/431)*
<!-- Sent from Reviewable.io -->

aledj2 commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 9 of 9 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/woook/moka-fe/431#-:-L2UPHDEAiBcWtOvmA75:bnfp4nl)*
<!-- Sent from Reviewable.io -->
